command/agent/auth/aws/aws.go

```diff
@@ -107,6 +134,27 @@ func NewAWSAuthMethod(conf *auth.AuthConfig) (auth.AuthMethod, error) {
 		}
 	}
 
+	if a.authType == typeIAM {
+
+		// Check for an optional custom frequency at which we should poll for creds.
+		credCheckFreqSec := defaultCredCheckFreqSeconds
+		if checkFreqRaw, ok := conf.Config["cred_check_freq_seconds"]; ok {
```

Comment:
Can we rename this to something else? This sounds complicated. Maybe `credential_poll_interval` or something?

Reply:
Well, I would say yes unhesitatingly, but I'm afraid that in a previous, separate PR for alicloud, I already instituted this variable name. https://github.com/hashicorp/vault/blob/master/command/agent/auth/alicloud/alicloud.go#L63

I'd be happy to do it if we're fine with them not being identical. It's a usability trade-off.

Reply:
Well, although I don't have a strong opinion on this, is it too late to rename this variable in here and for alicloud auth?

Reply:
I don't know . . . I doubt many people are using AliCloud auth yet? Probably not too late? It's only been out for a minute and probably nobody is using it yet, or very few people. I do _like_ `credential_poll_interval` better.

I will go ahead and update it and simultaneously do a separate PR for AliCloud auth to update it. I think the impact will be low or nil.

Reply:
https://github.com/hashicorp/vault/pull/5315